pulseaudio-control.bash

```diff
@@ -48,8 +48,16 @@ function getCurVol() {
 
```

Comment:
Now that I remember, the placeholder values for `SINK_BLACKLIST` and `SINK_NICKNAMES` at the top are now wrong. They should include the port, too, like this:

```sh
SINK_BLACKLIST=(
    "alsa_output.usb-SinkYouDontUse-00.analog-stereo/analog-output-lineout"
)

# ...

SINK_NICKNAMES["alsa_output.usb-SomeManufacturer_SomeUsbSoundcard-00.analog-stereo/analog-output"]="External Soundcard"
```